### DISCUSSION TITLE
shape='box' Add rigid body, why my rigid body is bigger than the actual object, I don't know what's wrong.

### DISCUSSION THREAD

**Discussion by volcanoqq** [enable3d/enable3d#364] [Help / Questions]
![image](https://github.com/enable3d/enable3d/assets/113505790/d6547a32-478e-4576-8111-66636bcd4c75)

And then I manually pass in width,height,depth, and it works.
![image](https://github.com/enable3d/enable3d/assets/113505790/cc66f1f2-5425-4d61-b114-d0fef70ce336)

I rotated the object in three.js before adding the rigid body, and then added the rigid body in the physical world in a box way, the rigid body did not apply the rotation, I don't know what went wrong
![image](https://github.com/enable3d/enable3d/assets/113505790/7da78215-ad89-41fe-b5aa-068113e6fd38)

**Comment by yandeu** [comment]
This is a very strange way of creating an box. Why don't you just use `this.physics.add.box()`?

**Reply by volcanoqq** [reply]
This is the model that I imported from the outside and I want to add physical collisions to this model

**Reply by volcanoqq** [reply]
![image](https://github.com/enable3d/enable3d/assets/113505790/6648ea24-9c5b-41c7-bebb-ec26e8f45041)

**Reply by volcanoqq** [reply]
why my rigid body is bigger than the actual object
